### ISSUE TITLE
Failed to configtest and scan

### ISSUE THREAD

**Issue by jiulingqiqi9077** [future-architect/vuls#1631]
# What did you do? (required. The issue will be **closed** when not provided.)
Scan other target server but failed.

# What did you expect to happen?
Scan successfully

# What happened instead?
Get error message when run the scan or configtest comands 

* Current Output
[root@vuls vulsctl-master]# vuls configtest 
[Mar 30 15:57:49]  INFO [localhost] vuls-v0.22.2-build-20230321_234726_7f35f4e
[Mar 30 15:57:49]  INFO [localhost] Validating config...
[Mar 30 15:57:49]  INFO [localhost] Detecting Server/Container OS... 
[Mar 30 15:57:49]  INFO [localhost] Detecting OS of servers... 
[Mar 30 15:57:49] ERROR [localhost] (1/1) Failed: remotehost, err: [Failed to print SSH configuration. err:
    github.com/future-architect/vuls/scanner.validateSSHConfig
        /root/go/src/github.com/future-architect/vuls/scanner/scanner.go:354
  - exit status 255]
[Mar 30 15:57:49] ERROR [localhost] Failed to configtest: Failed to init servers. err:
    github.com/future-architect/vuls/scanner.Scanner.Configtest
        /root/go/src/github.com/future-architect/vuls/scanner/scanner.go:115
  - No scannable host OS:
    github.com/future-architect/vuls/scanner.Scanner.initServers
        /root/go/src/github.com/future-architect/vuls/scanner/scanner.go:253

Please re-run the command using ```-debug``` and provide the output below.
[root@vuls vulsctl-master]# vuls configtest --debug
[Mar 30 15:58:22]  INFO [localhost] vuls-v0.22.2-build-20230321_234726_7f35f4e
[Mar 30 15:58:22]  INFO [localhost] Validating config...
[Mar 30 15:58:22]  INFO [localhost] Detecting Server/Container OS... 
[Mar 30 15:58:22]  INFO [localhost] Detecting OS of servers... 
[Mar 30 15:58:22] DEBUG [localhost] Validating SSH Settings for Server:remotehost ...
[Mar 30 15:58:22] DEBUG [localhost] Executing... /usr/bin/ssh -G -i /root/.ssh/id_rsa -p 22 -l root 10.236.19.217
[Mar 30 15:58:22] ERROR [localhost] (1/1) Failed: remotehost, err: [Failed to print SSH configuration. err:
    github.com/future-architect/vuls/scanner.validateSSHConfig
        /root/go/src/github.com/future-architect/vuls/scanner/scanner.go:354
  - exit status 255]
[Mar 30 15:58:22] ERROR [localhost] Failed to configtest: Failed to init servers. err:
    github.com/future-architect/vuls/scanner.Scanner.Configtest
        /root/go/src/github.com/future-architect/vuls/scanner/scanner.go:115
  - No scannable host OS:
    github.com/future-architect/vuls/scanner.Scanner.initServers
        /root/go/src/github.com/future-architect/vuls/scanner/scanner.go:253
# Steps to reproduce the behaviour

# Configuration (**MUST** fill this out):

* Go version (`go version`):
[root@vuls vulsctl-master]# go version
go version go1.20.2 linux/amd64

* Go environment (`go env`):
[root@vuls vulsctl-master]# go env
GO111MODULE=""
GOARCH="amd64"
GOBIN=""
GOCACHE="/root/.cache/go-build"
GOENV="/root/.config/go/env"
GOEXE=""
GOEXPERIMENT=""
GOFLAGS=""
GOHOSTARCH="amd64"
GOHOSTOS="linux"
GOINSECURE=""
GOMODCACHE="/root/go/pkg/mod"
GONOPROXY=""
GONOSUMDB=""
GOOS="linux"
GOPATH="/root/go"
GOPRIVATE=""
GOPROXY="https://proxy.golang.org,direct"
GOROOT="/usr/local/go"
GOSUMDB="sum.golang.org"
GOTMPDIR=""
GOTOOLDIR="/usr/local/go/pkg/tool/linux_amd64"
GOVCS=""
GOVERSION="go1.20.2"
GCCGO="gccgo"
GOAMD64="v1"
AR="ar"
CC="gcc"
CXX="g++"
CGO_ENABLED="1"
GOMOD="/dev/null"
GOWORK=""
CGO_CFLAGS="-O2 -g"
CGO_CPPFLAGS=""
CGO_CXXFLAGS="-O2 -g"
CGO_FFLAGS="-O2 -g"
CGO_LDFLAGS="-O2 -g"
PKG_CONFIG="pkg-config"
GOGCCFLAGS="-fPIC -m64 -pthread -Wl,--no-gc-sections -fmessage-length=0 -fdebug-prefix-map=/tmp/go-build1431936855=/tmp/go-build -gno-record-gcc-switches"

* Vuls environment:

Hash : ____

To check the commit hash of HEAD
$ vuls -v
[root@vuls vulsctl-master]# vuls -v
vuls-v0.22.2-build-20230321_234726_7f35f4e

or

$ cd $GOPATH/src/github.com/future-architect/vuls 
$ git rev-parse --short HEAD 

* config.toml:
[servers]

[servers.remotehost]
host               = "10.236.19.217"
user               = "root"
port               = "22"
keyPath            = "/root/.ssh/id_rsa"
scanMode           = ["fast"]

* command:

**Comment by MaineK00n** [comment]
Does the following command print SSH Configuration?
```console
$ /usr/bin/ssh -G -i /root/.ssh/id_rsa -p 22 -l root 10.236.19.217
```

**Comment by jiulingqiqi9077** [comment]
> Does the following command print SSH Configuration?
> 
> ```
> $ /usr/bin/ssh -G -i /root/.ssh/id_rsa -p 22 -l root 10.236.19.217
> ```

No, it says the --G is unknown option.

**Comment by MaineK00n** [comment]
What is your version of SSH?
```
$ /use/bin/ssh -V
```

**Comment by jiulingqiqi9077** [comment]
> What is your version of SSH?
> 
> ```
> $ /usr/bin/ssh -V
> ```

OpenSSH_6.6.1p1, OpenSSL 1.0.1e-fips 11 Feb 2013

**Comment by jiulingqiqi9077** [comment]
After upgrading openssh to version 7.4p1, it works well.

**Comment by mozqnet** [comment]
As the `-G` option seems to have been released around OpenSSH 6.8, the version of 6.6 is not implemented may be.
